.teamcity/_self/projects/WPComTests.kt

```diff
@@ -330,6 +331,18 @@ fun jetpackBuildType(screenSize: String): BuildType {
 	}
 }
 
+private object GutenbergPlaywrightE2E: BuildType ({
+	name = "Playwright WPCOM E2E Tests (desktop)"
+	description = "Runs Gutenberg E2E tests in desktop size using Playwright"
+
+    name = "Hello world"
+    steps {
+        script {
```

Comment:
```suggestion
        bashNodeScript {
```

TC compilation is failing because `script` hasn't been imported. I think you could get it to work with `bashNodeScript` instead, which is already imported

Reply:
@noahtallen Is there a way for me to verify the configuration without having to push it up? I've noticed you and @scinos both talking about TC compilation; how could I do this?

Reply:
It is possible, but it's kinda hard to set up. We haven't found a reproducible setup yet. Basically, you need to install IntelliJ IDEA from JetBrains, and then do some hacking to get it to download some kotlin files from TeamCity through the proxy. At that point, you can run builds and get syntax highlighting. It took me a few hours to set up last time, and I don't think it works any more :/

So it might be easier to just push, and then failures are reported by TeamCity pretty quickly in the build log (`Kotlin DSL compilation errors`)

Reply:
Gotcha, thanks. I've changed it back to `bashNodeScript` in ff08f3dcfc61d6a3a469c3cd011231e91d62254a.